setup.py

```diff
@@ -33,8 +33,8 @@
         'cachecontrol[filecache]',
         'defusedxml',
         'lockfile',
-        'jinja2<=2.11.3; python_version <= "2.7"',
-        'jinja2; python_version > "2.7"',
+        'jinja2<=2.11.3; python_version <= "3.6"',
```

Comment:
To minimize the effect, we'd better do not change the requirement for python27. 
Better modify it in this way:

```diff
         'jinja2<=2.11.3; python_version <= "2.7"',
-        'jinja2; python_version > "2.7"',
+        'jinja2; python_version <= "3.6"',
+        'jinja2>3.1.6; python_version > "3.6"',
```